tests/check/check_h26xsigned_auth.c

```diff
@@ -1761,7 +1760,7 @@ static void validation_helper_function(signed_video_t *sv, nalu_list_item_t *sei
     if (public_key_present) {
       ck_assert(auth_report);
       latest = &(auth_report->latest_validation);
-      ck_assert(latest);
+      ck_assert(latest->public_key_has_changed == wrong_key);
```

Comment:
You need to keep `ck_assert(latest);`, otherwise you can get a segmentation fault.
Hence, you should have both. This line replaces lines 1766 and 1769 below.